packages/core/test/test_utils.js

```diff
@@ -56,6 +56,6 @@ export function describeRepeated(title, fn) {
 
 export function submitForm(node) {
   act(() => {
-    node.querySelector("button[type=submit]").click();
+    Simulate.submit(node);
```

Comment:
This is due to the same regression with buttons no longer sending submit events as mentioned above https://github.com/jsdom/jsdom/issues/3117